Explorer-Release-Setup-Instructions.md

```diff
@@ -86,16 +87,21 @@ _For Raspberry Pi and similar ARM-based platforms use:_
 
 ```bash
 $ docker run -d --restart unless-stopped -p 28967:28967 \
-    -e WALLET="" \
-    -e EMAIL="" \
-    -e ADDRESS="" \
+    -e WALLET="0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" \
+    -e EMAIL="user@email.corp" \
+    -e ADDRESS="domain.ddns.net:28967" \
     -e BANDWIDTH="2TB" \
     -e STORAGE="2TB" \
     -v "<identity-dir>":/app/identity \
     -v "<storage-dir>":/app/config \
     --name storagenode storjlabs/storagenode:arm
 ```
 
+_For Windows Operatins Systems use the following line:_
+```bash
+$ docker run -d --restart unless-stopped -p 28967:28967 -e WALLET="0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" -e EMAIL="user@email.corp" -e ADDRESS="domain.ddns.net:28967" -e BANDWIDTH="2TB" -e STORAGE="2TB" -v "D:\identity\storagenode\":/app/identity -v "D:\data\":/app/config --name storagenode storjlabs/storagenode:alpha
```

Comment:
I think you don't want hardcoded paths here

Reply:
I wanted to give an example, because many users on Windows had trouble with it (due to backslash instead of forwardshlash)
I could add it as note?

Reply:
Changed it do reflect this as a note